cli/script/command-executor.ts

```diff
@@ -1223,6 +1364,43 @@ function requestAccessToken(): Promise<string> {
     });
 }
 
+export var runReactNativeBundleCommand = (entryFile: string, outputFolder: string, platform: string, sourcemapOutput: string): Promise<void> => {
+    var reactNativeBundleCommandArgs = [
```

Comment:
Controlling the packager directly seems error prone.. We'd be duplicating logic from `react-native/local-cli`. Could we just take a direct dependency on `react-native-cli` and use node to run it like how we did for the vsts task?